### DISCUSSION TITLE
LEDランド剥がれの対応

### DISCUSSION THREAD

**Discussion by togepy1662** [Yowkees/keyball#205] [Q&A]
keyball44において、LEDのランドが4つとも剥がれてしまいました。(no1,2)
この場合、no1はpro microとどう繋ぐべきなのか、no2はno1,3とどう繋ぐべきか教えていただけないでしょうか？(過去ディスカッションで、DIN,DOUTの繋ぎ方は理解しております)

![image](https://user-images.githubusercontent.com/35281310/229276498-7e38b749-0bb8-4b78-abcb-92d14935631e.jpeg)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
すみません。写真からはどのランドが剥がれているのか判断が難しいです。
下記にKeyball44の基板データを公開していますので、こちらで線を辿っていただき、接続することはできますでしょうか。
https://github.com/Yowkees/keyball/tree/main/keyball44/design_data
お手数ですがご確認お願いいたします。
